### PR TITLE
Do not install development dependencies in built docker image

### PR DESCRIPTION
**Pull request by kierenevans**
Avoid installing development dependencies in the console docker image, which would then increase the time taken to get copied to the php-fpm docker image, and time to push to the docker registry.

For magento2, there may be development extensions that get brought in with composer when installing the development tools at runtime, so need to run setup:upgrade and recompile the DI container.

For drupal8, we need to get phpunit installed into the php-fpm container for simpletest to work, but that means installing composer in the container or copying it across from the console container after installing. Decided to go with copying from console.

There is also a fix for checking if the containers are running for static mode, which has been broken since https://github.com/inviqa/harness-base-php/commit/0179b92768e8095c845eb1bd55cfad28bc084c12#diff-098f6bcd4621d373cade4e832627b4f6R20

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -83,9 +88,13 @@ function run_pipeline_tests()
     # Check if we have a composer.json so we can run tests
     # Check inside the console container as static builds do not
     # update the local filesystem
-    if ws exec stat composer.json > /dev/null 2>&1; then
+    if ws exec stat /app/composer.json; then
```
Comment (andytson-inviqa) [inline test]: why is the silencing of output removed?

Comment (kierenevans) [reply test]: Was for debugging as it wasn't firing the tests every time. Fixed :+1:

**Comment by kierenevans** [comment]
Build failing due to:
`■■■ > composer install --no-interaction --no-dev --optimize-autoloader` printing out the composer help.

I guess we need #351 merged in first for the new passthru work.

File: src/spryker/application/skeleton/composer-harness.json [hunk 2]
```diff
@@ -30,9 +30,12 @@
       "@test-unit",
       "@test-acceptance"
     ],
-    "test-quality": [
+    "devtools-setup": [
```
Comment (andytson-inviqa) [inline src/spryker/application/skeleton/composer-harness.json]: would `generate` be a better name?

Comment (andytson-inviqa) [reply src/spryker/application/skeleton/composer-harness.json]: (similiar in terminology as `go generate`)

Comment (kierenevans) [reply src/spryker/application/skeleton/composer-harness.json]: Updated :+1:

**Review by andytson-inviqa — APPROVED** [review]

**Pull request merged**